lib/resources/dstu2/document_reference.yaml

```diff
@@ -86,11 +82,12 @@ fields:
         }
       ]
     }
-  note: If author is not provided, the author will be determined based on the access token.
+  note: At most one practitioner can be populated.
```

Comment:
prefer to implement multiple notes via an unordered list. reference: https://github.com/cerner/fhir.cerner.com/blob/881ebafa32dad7f57c4985f6799edb46ec8e81aa/lib/resources/r4/patient_patch.yaml#L33

Reply:
[listed](https://github.com/cerner/fhir.cerner.com/pull/374/commits/6fb0b4a5a1bdcd0b07e086f542d44ded83b12430?file-filters%5B%5D=.yaml#diff-eefc10b8b4a495f72e678cead5851448L85)